.github/workflows/build.yaml

```diff
@@ -48,11 +68,11 @@ jobs:
     name: "python=${{ matrix.python-version }} os=${{ matrix.os }}"
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
       - name: install tox
-        run: python -m pip install -U tox
+        run: pip install tox
```

Comment:
Dropping the `-U`, in contrast with `python -m pip` vs `pip`, changes the meaning of this command.

Whether or not it's _wise_ to `pip install -U` in a build or not is something I'm happy to debate, but this is a big part of why I don't like seeing unrelated changes slipped into a PR. It's all too easy to accidentally change something which was done on purpose. Let's make sure we leave these lines alone in any refinement of this PR.